cmd/data.go

```diff
@@ -72,9 +74,9 @@ func (cmd *DataCmd) SetFlags(f *flag.FlagSet) {
 	f.StringVar(&cmd.sessionJSON, "session", "", "Specifies the file we restore session state from")
 	f.StringVar(&cmd.target, "target", "Spanner", "Specifies the target DB, defaults to Spanner (accepted values: `Spanner`)")
 	f.StringVar(&cmd.targetProfile, "target-profile", "", "Flag for specifying connection profile for target database e.g., \"dialect=postgresql\"")
-	f.BoolVar(&cmd.skipForeignKeys, "skip-foreign-keys", false, "Skip creating foreign keys after data migration is complete (ddl statements for foreign keys can still be found in the downloaded schema.ddl.txt file and the same can be applied separately)")
```

Comment:
`--skip-foreign-keys` in the data command was I guess used in Spanner db creation. Now since creation responsibility is shifted to schema command, please port this flag over to the `schema` command.

Reply:
since we are adding the foreign key as part of data.go so retained this flag as it might make sense in this case